chain/txValidator.go

```diff
@@ -89,10 +88,6 @@ func CheckTransactionAttribute(txn *transaction.Transaction) error {
 	return nil
 }
 
-func checkAmountPrecise(amount Fixed64, precision byte) bool {
-	return amount.GetData()%int64(math.Pow(10, 8-float64(precision))) != 0
-}
-
```

Comment:
Why get rid of precise check?

Reply:
Because the precise check is doing nothing (checking int % 1 == 0)... You can verify that...

Reply:
It is meaningful for others asset which precision < 8.